src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -88,6 +102,40 @@ bool DebugKeepNode
 ) : EntityBase();
 
 
+public partial record ProxyForward
+(
+	[PartitionKey] string Region,
+	[RowKey] int DstPort,
+    int SrcPort,
+	string DstIp
+) : EntityBase();
+
+public partial record ProxyConfig 
+(	
+	string Url,
```

Comment:
We can make this a URI? https://docs.microsoft.com/en-us/dotnet/api/system.uri?view=net-6.0